src/Microsoft.ML.TimeSeries/STL/InnerStl.cs

```diff
@@ -0,0 +1,321 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.TimeSeries
+{
+    internal class InnerStl
+    {
+        private readonly bool _isTemporal;
+        private double[] _seasonalComponent;
+        private double[] _trendComponent;
+        private double[] _residual;
+
+        // arrays for intermediate results
+        private List<double>[] _cycleSubSeries;
+        private List<double>[] _smoothedSubseries;
+
+        private double[] _s;
+        private double[] _t;
+        private double[] _detrendedY;
+        private double[] _c;
+        private double[] _deseasonSeries;
+
+        /// <summary>
+        /// the smoothing parameter for the seasonal component.
```

Comment:
suggestion: capitalize here too

Reply:
Updated.

---
In reply to: [442523913](https://github.com/dotnet/machinelearning/pull/5202#discussion_r442523913) [](ancestors = 442523913)